omod/src/main/resources/config.xml

```diff
@@ -20,7 +20,41 @@
         <defaultValue>/spa</defaultValue>
         <description>The base URL from where Single Page Applications are served</description>
     </globalProperty>
-    
+
+    <globalProperty>
+        <property>spa.frontend.resourceBaseUrl</property>
+        <defaultValue>/frontend</defaultValue>
+        <description>The base URL from where Single Page Applications static resources are served</description>
+    </globalProperty>
+
+    <globalProperty>
+        <property>spa.frontend.directory</property>
+        <defaultValue>frontend</defaultValue>
+        <description>The directory that holds static files for the single-spa</description>
+    </globalProperty>
+
+    <!-- serve static files for the frontend on /frontend url pattern -->
+    <filter>
+        <filter-name>sparesourcefilter</filter-name>
+        <servlet-name>spaResources</servlet-name>
+        <filter-class>org.openmrs.module.spa.filter.SpaResourcesFilter</filter-class>
+    </filter>
+    <filter-mapping>
+        <filter-name>sparesourcefilter</filter-name>
+        <url-pattern>/frontend/*</url-pattern>
+    </filter-mapping>
+
+
+    <servlet>
+        <servlet-name>spaResources</servlet-name>
+        <servlet-class>org.openmrs.module.spa.servlet.SpaResourcesServlet</servlet-class>
+        <!-- Don't "load-on-startup" in case initial setup wizard is needed -->
+    </servlet>
+    <servlet-mapping>
```

Comment:
Module servlets do not have mappings because they are accessed as CONTEXT_PATH/moduleServlet/moduleid/servletName

Reply:
https://wiki.openmrs.org/display/docs/Module+Servlets

Reply:
Thanks for the guide Daniel